app/assets/javascripts/map_marker.js.coffee

```diff
@@ -11,11 +11,13 @@ window.map_marker = (map, options = {}) ->
       fillOpacity: 0.35
       center: pos
       radius: options.radius
+    return marker
   else
-    new google.maps.Marker
+    marker = new google.maps.Marker
       map: map
       position: pos
       title: 'Recorded Location'
+    return marker
```

Comment:
CoffeeScript always returns, so you can skip the variable assignment and return if you want to. :smile: